Sources/Commands/SwiftTestCommand.swift

```diff
@@ -610,10 +610,10 @@ public struct SwiftTestCommand: AsyncSwiftCommand {
 }
 
 extension SwiftTestCommand {
-    func printCodeCovPath(_ swiftCommandState: SwiftCommandState) throws {
+    func printCodeCovPath(_ swiftCommandState: SwiftCommandState) async throws {
         let workspace = try swiftCommandState.getActiveWorkspace()
         let root = try swiftCommandState.getWorkspaceRoot()
-        let rootManifests = try temp_await {
+        let rootManifests = try await safe_async {
```

Comment:
My concern here is the same as on previous PRs: `safe_async` uses `DispatchQueue.sharedConcurrent` under the hood, which can cause thread explosion. IMO instead of using that we need a serial queue for manifest loading, or `OperationQueue` with a fixed number of threads.

Reply:
My concern is that the workspace make use of temp_await which blocks threads so if we call into it from a swift concurrency pool we run the risk of blocking a thread. I know that GCD risks thread explosion, but that is less bad than a deadlocked thread IMO.  At the root of this tension is the integration into the LLBuild system which does not support async/await or callbacks.  I think I have a plan for how I can limit the unsafe blocking behavior to GCD queue dedicated to builds. Let me take some time to revisit this and make it more holistically address this